modules/clear-fields.js

```diff
@@ -0,0 +1,5 @@
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
- [ ] Please note that Linters analyzes source code to flag programming errors, bugs, stylistic errors, and suspicious constructs and it will make sure that your code looks clean, organized, and readable,  Kindly Remove the linters disable commands and try to fix the errors 🙏